### DISCUSSION TITLE
Websocket version of the API

### DISCUSSION THREAD

**Discussion by thatrandomperson5** [BattlesnakeOfficial/feedback#238] [Feature Requests]
With many, fast requests coming to the snakes all at once, it would seem better to use something designed for this like web sockets. An example packet format would be something like this:
```json
{
  "event": "move"
  "data": {...}
}
```
This would make the snakes more efficient, and less resource intensive.

**Comment by bvanvugt** [comment]
Definitely a long-standing request, there may be a duplicate of this somewhere...

**Reply by thatrandomperson5** [reply]
I searched and could not find one, this is why I made a new one.
